docker/webui/nginx-entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+
+replicas=""
+for i in $(seq 2 $replicas); do
+  replicas="server webui_webui_$i:9526;$replicas"
```

Comment:
The will override itself for more than one replica.

Reply:
Yes, the idea is to create a list of all webui_webui_1, 2, 3...  and this works

Reply:
Ah, sorry, I didn't look properly